shared/appengine.go

```diff
@@ -100,7 +100,7 @@ func (a appEngineAPIImpl) GetHTTPClientWithTimeout(timeout time.Duration) *http.
 
 func (a *appEngineAPIImpl) GetGitHubClient() (*github.Client, error) {
```

Comment:
nit: should we change `GetGitHubClient` to `NewGitHubClient`? Same goes for `GetGitHubClientFromToken`? Although most of the functions in this file start with `Get*`

Reply:
Renamed `GetGitHubClientFromToken` to `NewGitHubClientFromToken`, but decided to keep `GetGitHubClient` because it's not a constructor but an actual getter.